extension/manifest.json

```diff
@@ -407,6 +407,8 @@
         "https://www.google.co.zm/*",
         "https://www.google.co.zw/",
         "https://www.google.co.zw/*",
+        "https://encrypted.google.com/*",
+        "https://encrypted.google.com/",
```

Comment:
It is not necessary to have both `https://example.com/` and `https://example.com/*`. You can reduce the number of match patterns by about 50% by removing the match patterns without wildcard (and keep those with the wildcard).